src/components/PersonLink.tsx

```diff
@@ -0,0 +1,16 @@
+import cn from 'classnames';
+import { Link } from 'react-router-dom';
+import { Person } from '../types';
+
+export const PersonLink = ({ person }: { person: Person }) => {
+  return (
+    <Link
+      to={`./${person.slug}`}
+      className={cn({
+        'has-text-danger': person.sex === 'f',
+      })}
+    >
+      {person.name}
+    </Link>
+  );
+};
```

Comment:
lets do some structure refactoring

create two folders:

1. `pages`: put here 'HomePage', 'PeoplePage'
2. `components`: put here all other components we use for our pages like 'Error', 'PersonLink' etc